gobblin-compaction/src/main/java/gobblin/compaction/verify/CompactionVerifier.java

```diff
@@ -7,7 +7,8 @@
  */
 public interface CompactionVerifier<D extends Dataset> {
    String COMPACTION_VERIFIER_PREFIX = "compaction-verifier-";
-
+   String VERIFICATION_TIMEOUT = "compaction.verification.timeout";
```

Comment:
It's a good idea to include the units, e.g. `compaction.verification.timeoutMinutes`